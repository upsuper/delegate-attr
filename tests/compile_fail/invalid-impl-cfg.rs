use delegate_attr::delegate;

struct Foo(Vec<u8>);

#[delegate(x)]
#[cfg(not(fake))]
impl Foo {
    x
}

fn main() {}
