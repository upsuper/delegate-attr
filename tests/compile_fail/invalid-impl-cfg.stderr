error: expected one of `!` or `::`, found `}`
 --> $DIR/invalid-impl-cfg.rs:9:1
  |
7 | impl Foo {
  |          - while parsing this item list starting here
8 |     x
  |      - expected one of `!` or `::`
9 | }
  | ^
  | |
  | unexpected token
  | the item list ends here

error: invalid impl block for #[delegate]
 --> $DIR/invalid-impl-cfg.rs:7:1
  |
7 | impl Foo {
  | ^^^^
